examples/fscloud/main.tf

```diff
@@ -101,33 +100,19 @@ module "vpc" {
 }
 
 ########################################################################################################################
-# Observability Instances (Sysdig + AT)
+# Observability Instances (Cloud Monitoring)
 ########################################################################################################################
 
-locals {
-  existing_at = var.existing_at_instance_crn != null ? true : false
-  at_crn      = var.existing_at_instance_crn == null ? module.observability_instances.activity_tracker_crn : var.existing_at_instance_crn
-}
 
-
-# Create Sysdig and Activity Tracker instance
+# Create Cloud Monitoring instance
 module "observability_instances" {
-  source  = "terraform-ibm-modules/observability-instances/ibm"
-  version = "2.18.1"
-  providers = {
-    logdna.at = logdna.at
-    logdna.ld = logdna.ld
-  }
+  source                         = "terraform-ibm-modules/observability-instances/ibm"
+  version                        = "3.0.0"
   region                         = var.region
   resource_group_id              = module.resource_group.resource_group_id
   cloud_monitoring_instance_name = "${var.prefix}-sysdig"
   cloud_monitoring_plan          = "graduated-tier"
-  enable_platform_logs           = false
   enable_platform_metrics        = false
-  log_analysis_provision         = false
-  activity_tracker_instance_name = "${var.prefix}-at"
-  activity_tracker_plan          = "7-day"
-  activity_tracker_provision     = !local.existing_at
   cloud_logs_provision           = false
```

Comment:
This needs to be set to `true`.
and the [logs agents](https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/variables.tf#L233) should send logs to the Cloud logs.

Reply:
same as https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/pull/542/files#r1800842243